cypress/integration/regressions.spec.js

```diff
@@ -129,13 +120,6 @@ context('Regressions', () => {
     )
   })
 
-  it('The Cree word ayiwinis should give you ayiwinisa as lemma', () => {
```

Comment:
these "idiomatic lemma" test doesn't make sense anymore, so we can just delete them. We had the issue cuz we used a lemma_tags.tsv as our lemma form guideline, (which say, decides the lemma of NI words should be N+I+Sg). The importing process has been changed and the lemma form depends solely on the fst we have.